packages/flutter/lib/src/cupertino/page.dart

```diff
@@ -0,0 +1,137 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+
+const double _kMinFlingVelocity = 1.0;  // screen width per second.
+const Color _kBackgroundColor = const Color(0xFFEFEFF4); // iOS 10 background color.
+
+// Used for iOS.
+class CupertinoPageTransition extends AnimatedWidget {
+  CupertinoPageTransition({
+    Key key,
+    Animation<double> animation,
+    this.child,
+  }) : super(
+    key: key,
+    listenable: _kTween.animate(new CurvedAnimation(
+      parent: animation,
+      curve: new _CupertinoTransitionCurve(null),
+    ),
+  ));
+
+  static final FractionalOffsetTween _kTween = new FractionalOffsetTween(
+    begin: FractionalOffset.topRight,
+    end: -FractionalOffset.topRight,
+  );
```

Comment:
We usually make these file-level statics for no particular reason.  This is fine too.